### ISSUE TITLE
Standalone BatchNorm Conversion for Torch and Keras

### ISSUE THREAD

**Issue by quic-ristha** [quic/aimet#2136]
Sandalone batchnorm parameters need to be converted  for Torch and Keras Framework
